projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -386,9 +386,23 @@ protected function __construct() {
 		wp_register_style( 'grunion.css', GRUNION_PLUGIN_URL . 'css/grunion.css', array(), JETPACK__VERSION );
 		wp_style_add_data( 'grunion.css', 'rtl', 'replace' );
 
+		self::enqueue_contact_forms_style_script();
 		self::register_contact_form_blocks();
 	}
 
+	/**
+	 * Enqueue scripts responsible for handling contact form styles.
+	 */
+	private static function enqueue_contact_forms_style_script() {
+		wp_enqueue_script(
+			'contact-form-styles',
+			plugins_url( 'js/form-styles.js', __FILE__ ),
+			array( 'jquery' ),
+			JETPACK__VERSION,
+			true
+		);
+	}
```

Comment:
I think it would be best if we could find a solution so we don't need to enqueue 2 new JavaScript files, including the full jQuery, on all pages that have a contact form, even when we do not need that functionality.

In the past, we've done a lot of work to remove the use of jQuery in Jetpack ([reference](https://github.com/Automattic/jetpack/pulls?q=is%3Apr+jquery+is%3Aclosed+label%3A%22%5BFocus%5D+Performance%22)), so I would recommend not adding it to the frontend at all for the contact form.

Reply:
I think I can rewrite this script to remove the jQuery dependency.
What I didn't get was the `so we don't need to enqueue 2 new JavaScript files`. 
Is there any issue/concern about enqueuing new JavaScript files?

Reply:
We can definitely enqueue new files, no issues there. I would just recommend being mindful of the performance impact of anything new we enqueue on the frontend, especially for features like forms that may be added to every page on a website (via a block in the footer for example).

Reply:
Got it!
I've refactored the Javascript file to remove the jQuery dependency, but we still need this JS file to be loaded whenever a Form exists on the page.